doc/ref_arch/openstack/chapters/chapter05.md

```diff
@@ -162,10 +165,16 @@ The exhaustive list of extensions is available at https://docs.openstack.org/api
 
 
 <a name="5.3"></a>
-## 5.3. Consolidated Set of APIs.
+## 5.3. Consolidated Set of APIs
 
 ### 5.3.1. OpenStack Interfaces
-This section illustrates some of the Interfaces provided by OpenStack; the exhaustive list of APIs is available at https://docs.openstack.org/api-ref/ and CNTT standard is to use the latest version v3. OpenStack REST APIs are simple to interact with using either of two options. Clients can either call the APIs directly using the HTTP or REST library, or they can use one of the many programming language specific cloud libraries.
+This section illustrates some of the Interfaces provided by OpenStack; the exhaustive list of APIs is available 
+at https://docs.openstack.org/api-ref/. All the releases of OpenStack (>= Pike) that **get certified by RC** can be 
+considered as a CNTT RA Compliant. When more than one OpenStack release gets certified then the "CNTT standard" will
+ be the oldest CNTT RA compliant OpenStack release. Example, Let us assume that the Pike, Queens and Stein OpenStack 
+ releases get certified. Then the CNTT standard OpenStack version will be Pike. OpenStack REST APIs are simple to 
+ interact with using either of two options. Clients can either call the APIs directly using the HTTP or REST library,
+  or they can use one of the many programming language specific cloud libraries.
```

Comment:
@deebhatia Suggest breaking the paragraph -- move last 2 sentences "OpenStack REST APIs are simple to 
 interact with using either of two options. Clients can either call the APIs directly using the HTTP or REST library,
  or they can use one of the many programming language specific cloud libraries." to new paragraph.

Reply:
Done.